index.js

```diff
@@ -1,56 +1,52 @@
-'use strict';
-var childProcess = require('child_process');
-var neatCsv = require('neat-csv');
-var sec = require('sec');
-var pify = require('pify');
-var Promise = require('pinkie-promise');
-
-module.exports = function (opts) {
+const childProcess = require('child_process');
+const pify = require('pify');
+const neatCsv = require('neat-csv');
+const sec = require('sec');
+
+module.exports = opts => {
 	if (process.platform !== 'win32') {
 		return Promise.reject(new Error('Windows only'));
 	}
 
-	opts = opts || {};
+	const options = opts || {};
```

Comment:
Don't change this. I have it like this so it will be super easy to change to default parameters when I can target Node.js 6.